src/legal_management.md

```diff
@@ -0,0 +1,70 @@
+# Management
+
+This wiki page describes the bare minimum management tasks required to keep the space up and running. In general these tasks should be handled by the current board of directors.
+
+ALTSpace (Society of Air Light Time Space SPC) is a Social Purpose Corporation registered in the state of Washington with Federal Tax Exempt 501(c)(7) status.
+
+### Document Locations
+All important legal documents can be found in either the safe or the filing cabinet. Board members should have access to both. Yearly filings should all be kept in these locations.
+
+### Organizational Information
+* Registered Name: SOCIETY OF AIR LIGHT TIME SPACE SPC
+* Registered State: Washington
+* Entity Type: Social Purpose Corporation 
+* Federal IRS Status: Federal Tax Exempt 501(c)(7)
+* EIN & UBI can be found in the locked filing cabinet
+
+## Government Filing Requirements
+
+### Federal Taxes
+ALTSpace is Federal Tax Exempt by 501(c)(7), however, we still need to file a yearly report with the IRS.
+
+* Who: Current Treasurer
+* When: By May 15th
+* What: Form 990-EZ ([link](https://www.irs.gov/pub/irs-pdf/f990ez.pdf))
+
+### Washington State taxes
+* Who: Current Treasurer
+* When: By April 15th 
+* What: Combined Excise Tax Return - Can file through secure.dor.wa.gov
+* B&O classification is “Service and Other Activities; Gambling Contests of Chance”
+* Cost: Minimal, should be none if our revenue is < $50,000
+
+### Secretary of State annual report
+* Who: Current Secretary
+* When: June 30
+* What: Annual Report can be filed at https://www.sos.wa.gov/corporations-charities (Click “Express Annual Report”)
+* Cost: $60
+
+### City of Seattle requirements
+* We don’t file anything with the city, but possibly we’re supposed to?
+* Michael was going to look into this for us, not sure what he found out
```

Comment:
No need to throw michael under the bus on the wiki :p

I reached out to the city and set up an appointment for next tuesday to figure this out, so we can just leave this section out for now and update it in a week

Reply:
Yeah, mostly didn't want to loose context. Also I was hoping he'd review and comment if he'd figured it out :)

Reply:
I deserve to be thrown under that bus, but good looking out Noah. =)